src/components/SpecialResolution/Resolution.vue

```diff
@@ -34,7 +34,7 @@
             <v-col>
               <h2>Special Resolution</h2>
               <v-chip
-                v-if="isCoopCorrectionFiling && hasChanged"
+                v-if="hasChanged && isCoopCorrectionFiling"
```

Comment:
What's the difference? Could the first condition be False and we don't want to check the second one?

Reply:
No difference

Reply:
OK, no special comment needed then :)